<launch>
	<param name="/buidling_search_mission" value="2.0"/>
	<param name="/marker_mission" value="3.0"/>

	<node pkg="offboard" type="setmode_node.py" name="setmode_node" required="true" output="screen"/>

	<!-- <node pkg="building_search" type="building_search_node" name="building_search_node" required="true" output="screen"/> -->
	<node pkg="building_search" type="building_search_python_node.py" name="building_search_python_node" required="true" output="screen"/>
		<param name="search_height" value="8.1"/>
		<param name="building_search_count" value="15"/>
		<param name="building_stack_count" value="15"/>
		<param name="building_search_radius" value="0"/>
		<param name="building_search_speed" value="-0.02"/>

	
	<node pkg="safety_landing" type="aruco_VIO.py" name="aruco_VIO" output="screen" />

	<!--<node pkg="safety_landing" type="PID_control_node.py" name="PID_control_node" output="screen" />-->

	<!--<node pkg="safety_landing" type="RL_control_node.py" name="RL_control_node" output="screen"/>-->
	<node pkg="safety_landing" type="RL_aruco_control_node.py" name="RL_aruco_control_node" output="screen"/>
	<node pkg="koreauav_utils" type="cargo_launch.py" name="cargo_launch" output="screen"/>
		<param name="cargo_offset" value="1.0"/>

</launch>
